testing/ncids-css-testing/stories/uswds-native/usa-collection/src/index.scss

```diff
@@ -0,0 +1,5 @@
+@use "styles/ncids";
+@forward "uswds-fonts";
+@forward "uswds-utilities";
+@forward "usa-link";
+@forward "usa-collection";
```

Comment:
we need the usa-tag import here - but we'll also need to discuss accessibility issue with `..usa-collection__meta-item.usa-tag--new`

```scss
.usa-collection__meta-item {
  ...

  &.usa-tag {
   ...

    &--new {
      @include u-bg("accent-warm-dark");
      @include u-text("white");
    }
  }
}
```

Reply:
un resolving until we can get this to pass accessibility